server/database/db.js

```diff
@@ -9,12 +9,12 @@ const sequelize = new Sequelize('mydb', 'noobdev', 'post', {
 
 
 var biz = sequelize.define('business', {
-  "bizID" : Sequelize.TEXT,
+  "biz_iD" : Sequelize.TEXT,

```

Comment:
Did you intend for this to be biz_iD? Or did you mean biz_id?

Reply:
Also with these changes, do you need to change your seed.js to accommodate these new column headers?

Reply:
oops, I will change that one. Thanks.

Reply:
I don't have seed.js?